protocol/communities/manager.go

```diff
@@ -598,7 +602,11 @@ func (m *Manager) CreateCategory(request *requests.CreateCommunityCategory, publ
 	if community == nil {
 		return nil, nil, ErrOrgNotFound
 	}
+
 	categoryID := uuid.New().String()
+	if request.ThirdPartyID != "" {
+		categoryID = categoryID + request.ThirdPartyID
+	}
```

Comment:
@cammellos as we now process the import in multiple chunks/files, there's a possibility that we'll run into files that reference an already existing chat or category.

In order to check whether or not a given category/or channel already exists, we attach the discord category/chat ID to ours and then enable ourselves to check if any existing category/chat has a certain suffix.

Chat and category IDs are otherwise random IDs, so it's hard to check if we already have a chat/category otherwise (name alone isn't enough)